addons/pos_restaurant/models/pos_order.py

```diff
@@ -81,17 +100,7 @@ def _get_order_lines(self, orders):
         if order_lines != []:
             self._get_pack_lot_lines(order_lines)
 
-        extended_order_lines = []
-        for order_line in order_lines:
-            order_line['product_id'] = order_line['product_id'][0]
-            order_line['server_id'] = order_line['id']
-
-            del order_line['id']
-            if not 'pack_lot_ids' in order_line:
-                order_line['pack_lot_ids'] = []
-            else:
-                order_line['pack_lot_ids'] = [[0, 0, lot] for lot in order_line['pack_lot_ids']]
```

Comment:
It'll be better to just export this code to another function called (something like) `_prepare_order_lines(order_line)`  
and then do `extended_order_lines.append([0, 0, self._prepare_order_lines(order_line)`.

That way, you will never have to deal with the fact that it's a list of list (`[[0,0, order_line]] in the return during the override